### PR TITLE
created login page with website color scheme

### PR DESCRIPTION
**Pull request by sandilya27**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -183,6 +183,15 @@ html {
...
 
+.btn-link{
+  text-decoration: none;
+  color: #fff;
+}
+.btn-link:hover{
+  text-decoration: none;
+  color: #000;
+}
+
```
Comment (sandilya27) [inline style.css:194]: I forget to add .btn-link:focoused{color:#000;} please add it and accept my pull request

Comment (pythoniclaver) [reply style.css:194]: > I forget to add .btn-link:focoused{color:#000;} please add it and accept my pull request

Add it yourself and make another PR

Comment (pythoniclaver) [reply style.css:194]: > I forget to add .btn-link:focoused{color:#000;} please add it and accept my pull request

I just reviewed the project and I must admit, I love it. Nice job you did there man 💙

Comment (pythoniclaver) [reply style.css:194]: > I forget to add .btn-link:focoused{color:#000;} please add it and accept my pull request

@sandilya27 please when you make the changes let me know

Comment (sandilya27) [reply style.css:198]: @pythoniclaver changes are done please review it and accept the pull request

**Comment by pythoniclaver** [comment]
Thanks for contributing!

File: style.css (hunk 1 above)
Comment (pythoniclaver) [reply style.css:198]: @sandilya27 Our Repository deserve a star from you, don't we?😭😭😭

Drop a star so we would be motivated to  create another issue 🙏.

**Pull request merged**